docs/faq.md

```diff
@@ -3,7 +3,7 @@ title: FAQ
 layout: default
 ---
 
-**What your release cycle?**
+**What is your release cycle?**
```

Comment:
If a compiler can detect problems, why can't we make something that can scan a document for grammar problems? xD (project idea).

Reply:
see @ifmeorg/ifme-languagetools :)